scripts/biomedical/diseasesAtJensenLab/scripts/format_disease_jensen_lab.py

```diff
@@ -0,0 +1,250 @@
+    df['dcid'] = 'bio/' + df['Disease'].astype(str).str.replace(':', '_', regex=False) + '_' + df['Gene'].astype(str) + '_association'
+    # format name for DiseaseGeneAssociation nodes
+    df['name'] = df['Disease_Name'].astype(str) + ' and ' + df['Gene'].astype(str) + ' association'
+    # drop columns not used in import
+    df = df.drop(['Disease'], axis=1)
+    return df
+
+
+def format_RNA_type(df_tm):
+	gene_list = ['orf', 'ZNF']
+	sno_rna_list = ['sno', 'SNOR', 'SCAR']
+	linc_rna_list = ['LINC', 'linc', 'MIR']
+	df_tm['RNA_type'] = 'dcs:NonCodingRNATypeLongNonCodingRNA'
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('|'.join(gene_list)),'Gene', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('rRNA'),'dcs:NonCodingRNATypeRibosomalRNA', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('|'.join(linc_rna_list)),'dcs:NonCodingRNATypeLongIntergenicNonCodingRNA', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('|'.join(sno_rna_list)),'dcs:NonCodingRNATypeSmallNucleolarRNA', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('miR'),'dcs:NonCodingRNATypeMicroRNA', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('circ'),'dcs:NonCodingRNATypeCircularRNA', df_tm['RNA_type'])
+	df_tm['RNA_type'] = np.where(df_tm["Gene"].str.contains('pRNA'),'dcs:NonCodingRNATypePromoterAssociatedRNA', df_tm['RNA_type'])
+	return df_tm
+
+
+def format_dcids(df, data_type):
+	df = format_doid_icd(df)
+	df = format_disease_gene_cols(df, data_type)
+	df = df.apply(lambda x: check_for_dcid(x),axis=1)
+	return df 
+
+
+def format_data_type_specific_info(df, data_type):
+	if data_type == 'experiments':
+		df['associationType'] = 'dcs:AssociationSourceExperiment'
+		df['source-score'] = df['source-score'].str.split('=')
+		df['source-score'] = np.where(df['source-score'] == df['source-score'],df['source-score'].str[1],np.nan)
+		df.update('"' +
+				  df[['Disease_Name', 'score-db']].astype(str) + '"')
+	if data_type=='knowledge':
+		df['associationType'] = 'dcs:AssociationSourceKnowledge'
+		df.update('"' +
+				  df[['Disease_Name', 'score-db']].astype(str) + '"')
+	if data_type =='textmining':
+		df['associationType'] = 'dcs:AssociationSourceTextMining'
+		df.update('"' +
+				  df[['Disease_Name', 'url']].astype(str) + '"')
+	return df
+
+
+def clean_data(df, data_type):
+	df_tm = df
+	searchfor = ['ENSP00', 'LINC', 'linc'] ## filter out non coding RNAs
+	df = df[df['Id'].str.contains("ENSP00")]
+	df = df[~df.Gene.str.contains('|'.join(searchfor))]
+	df_tm = df_tm[~df_tm.isin(df)].dropna() ## df with only non coding RNAs
+	df_tm = df_tm[~df_tm['Gene'].str.contains("chr")]
+	df_tm = df_tm[~df_tm['Gene'].str.contains("ENSP00")]
+	df = format_dcids(df, data_type)
+	df_tm = format_dcids(df_tm, data_type)
+	df_tm = format_RNA_type(df_tm) ## filter out genes from df with non coding RNA
```

Comment:
nit: does the function format_RNA_type do filtering? if so, please add a comment on the function because that's not clear from the name of the function